arm-servicefabric/2016-09-01/swagger/servicefabric.json

```diff
@@ -388,14 +388,14 @@
       }
     },
     "SettingsSectionDescription": {
-      "description": "Settings section description",
+      "description": "ServiceFabric section settings",
       "properties": {
         "name": {
           "type": "string",
-          "description": "The name of settings section description"
+          "description": "The name of settings section"
         },
         "parameters": {
-          "description": "Settings parameter description",
+          "description": "Settings parameter value",
```

Comment:
Can we elaborate a a little here like "Collection of settings in the section, each setting is a tuple consisting of setting name and value"